Arcs/Arcs.cpp

```diff
@@ -504,6 +815,527 @@ std::pair<bool, bool> headOrTail(int head, int tail) {
     }
 }
 
+
+
+/* Shreds end sequence into kmers and inputs them one by one into the ContigKMap
+ * 	std::pair<std::string, bool> 				specifies contigID and head/tail
+ * 	std::string						the end sequence of the contig
+ *	int							k-value specified by user
+ *	ARKS::ContigKMap					ContigKMap for storage of kmers
+ */
+int mapKmers(std::string seqToKmerize, int k,
+		ARCS::ContigKMap &kmap, ReadsProcessor &proc, int conreci) {
+
+	int seqsize = seqToKmerize.length();
+
+	// Checks if length of the subsequence is smaller than the k size
+	// 	If the contig end is shorter than the k-value, then we ignore.
+	if (seqsize < k) {
+		std::string errmsg =
+				"Warning: ends of contig is shorter than k-value for contigID (no k-mers added): ";
+		std::cout << errmsg << conreci << std::endl;
+
+		return 0;
+	} else {
+		//assert(seqsize >= k);
```

Comment:
I know this is probably left over from ARKS, but delete any lines of commented-out code